pulser-core/pulser/sequence/_schedule.py

```diff
@@ -156,15 +158,18 @@ def get_samples(self) -> ChannelSamples:
             slots.append(_TargetSlot(s.ti, tf, s.targets))
 
             # The phase of detuned delays is not considered
```

Comment:
I guess this comment should be changed `# The phase of detuned delays can be not considered`